examples/python/rippleadd-async.py

```diff
@@ -120,24 +120,27 @@ def unmajority(p, a, b, c):
 qobjs = []
 # Create online (so I/O bound) jobs if we have connetion or local (so CPU bound)
 # jobs otherwise
-if offline == False:
+if offline is False:
     print("Creating %d online jobs..." % NUM_JOBS)
     for _ in range(0, NUM_JOBS):
+
         qobjs.append(qp.compile(["rippleadd"], backend=online_backend,
-                coupling_map=None, shots=1024))
+                                coupling_map=None, shots=1024))
+
+        print("Creating %d local jobs..." % NUM_JOBS)
+        # Create CPU intensive jobs
+        for _ in range(0, NUM_JOBS):
 
-print("Creating %d local jobs..." % NUM_JOBS)
-# Create CPU intensive jobs
-for _ in range(0, NUM_JOBS):
-    qobjs.append(qp.compile(["rippleadd"], backend=local_backend,
-              coupling_map=None, shots=1024))
+            qobjs.append(qp.compile(["rippleadd"], backend=local_backend,
+                                    coupling_map=None, shots=1024))
+        end = False
```

Comment:
Hmm, could you please revert this "block" of code (from the line after the `if not offline` up to this one) to the original version, changing _just_ the indentation of the second line of the `qobjs.append()` calls? The main reason is that the proposed changes in indentation actually modify the logic, making the `for` loops to be nested.